radixdlt-java/src/test/java/com/radixdlt/client/core/atoms/AtomTest.java

```diff
@@ -16,6 +19,7 @@ public void testEmptyAtom() {
 		assertTrue(atom.getDataParticles().isEmpty());
 		assertTrue(atom.getConsumables(Spin.UP).isEmpty());
 		assertTrue(atom.getConsumables(Spin.DOWN).isEmpty());
+		assertEquals("bf6a73657269616c697a65721a001ed1516776657273696f6e1864ff", ByteString.of(atom.toDson()).hex());
```

Comment:
Maybe this assertion would benefit from a short comment where this hex-hash comes from. Is it generated by the JS-library? Kotlin? Some external tool?

Reply:
@laszlourszuly sorry missed your review comments :) 

This is, currently, the value generated by this library. I.e. if we make changes in the Atom, it will result in new Hash and thus the test will need to get updated.